aws/rapids_ec2_mnmg.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    client.wait_for_workers(n_workers)
</pre><p>Yep. This would be <code>client.wait_for_workers(n_workers*n_gpus_per_worker)</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Huh, it still works and shows me the correct number of workers here. But I'll update it, thanks!</p>